distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -53,138 +53,164 @@ The default metacard type has the following required attributes.
 Though the following attributes are required on all metacard types, setting their values is optional except for ID.
```

Comment:
Where would I find out where we're injecting these if they're not present. I can check and verify if they'll all be injected if they aren't set explicitly.